gitlab-ls.py

```diff
@@ -210,8 +227,9 @@ def get_merge_request_dict(project: Project, updated_after: str = None) -> Dict[
                 title=mr.title,
                 author=mr.author["name"],
                 open=(mr.state == "opened"),
+                description=mr.description,
             )
-            if mr.iid ==886:
+            if mr.iid == 886:
```

Comment:
This is a debug that line that I forgot to cleanup. Could you ?

Reply:
done